resources/service.rb

```diff
@@ -0,0 +1,96 @@
+provides :nssm_service, platform: 'windows'
+# TODO: migrate to nssm_service with a breaking change notice
```

Comment:
Yes I think a v4.0.0 would be great, but currently I think my PR is not enough. Due to the way nssm binary is installed upgrade is not possible when a service is currently running.
In addition, the implementation I made for the `load_current_value` does not work with older version of `nssm`, another blocking point for the upgrade.

The `nssm_service` would be great if we start having a resource for `nssm_install` for instance.
I really think we should migrate the `install` recipe into a resource, to enhance install/upgrade scenarii and allow people to subscribe to that kind of event.